DOCS-7400 Added Use Cases to Kafka (#725)
mulesoft/docs-connectors: modules/ROOT/pages/kafka/kafka-connector-reference.adoc

Kafka Connector v4.0

> Anypoint Connector for Apache Kafka (Apache Kafka Connector) enables you to interact with the Apache Kafka messaging system, and provides seamless integration between your Mule app and an Apache Kafka cluster, using Mule runtime engine (Mule).

Reviewer: ```suggestion
Anypoint Connector for Apache Kafka (Apache Kafka Connector) enables you to interact with the Apache Kafka messaging system. It provides seamless integration between your Mule app and an Apache Kafka cluster, using Mule runtime engine (Mule).
```

|===
| Name | Type | Description | Default Value | Required
| Bootstrap server URLs a| Array of String |  The list of servers to bootstrap the connection with the Kafka cluster. This can be a partial list of the available servers. |  | x

> | Batch size a| Number |  The producer attempts to batch records together into fewer requests whenever multiple records are sent to the same partition. This helps performance on both the client and the server. This configuration controls the default batch size in bytes. No attempt is made to batch records larger than this size. Requests sent to brokers will contain multiple batches, one for each partition with the data that is available to send. A small batch size makes batching less common and can reduce throughput (a batch size of zero disables batching entirely). A very large batch size can result in more wasteful use of memory as a buffer of the specified batch size is always allocated in anticipation of additional records. |  `16` |

Reviewer: ```suggestion
| Batch size a| Number |  The producer attempts to batch records into fewer requests whenever multiple records are sent to one partition. This helps both client and server performance. This configuration controls the default batch size, in bytes. No attempt is made to batch records larger than this size. Requests sent to brokers contain multiple batches of available data, one for each partition. A small batch size makes batching less common and can reduce throughput (a batch size of zero disables batching entirely). A very large batch size can result in wasted memory, because a buffer of the specified batch size is always allocated in anticipation of additional records. |  `16` |
```
This is a lot of conceptual info for a table entry. Consider whether it would be better presented elsewhere. Look at the other multisentence descriptions as well.

Author: This is the reference doc. It's generated from code comments
